.gitignore

```diff
@@ -20,6 +20,9 @@
 *.pyc
 *.python-version
 
+# Ruby
```

Comment:
Does this need to be added it to the MM repository?

Reply:
When I pushed from my environment, it tried to modify the Ruby setup so I though I would add it in the ignore..

Reply:
I removed the file